Get Started/landing.css

```diff
@@ -0,0 +1,60 @@
+:root{
+    --primary-color: #FF6F61;
+    --black-text: #1E1C1C;
+}
+
+a{
+    color: var(--black-text);
+}
+
+.primary-highlight{
+    color: var(--primary-color) !important;
+}
+
+.nav-bar{
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 2rem 1rem;
+    background-color: var(--page-background);
+    padding-left: 5rem;
+    padding-right: 5rem;
+}
+.left-navbar{
+    cursor: pointer !important;
```

Comment:
avoid using !important . if you are using for the material icon then i can understand . But here you are using this for pointer , why????